src/install.js

```diff
@@ -73,11 +73,11 @@ function checkSetupEnv() {
 		NODEBB_DB_SSL: 'ssl',
 	};
 
-	Object.keys(evars).forEach((v) => {
-		if (v.match(envNbbRe)) {
-			setupVal[envopts[v]] = evars[v];
-		} else if (v.match(envNbbDbRe)) {
-			setupVal[`${evars.NODEBB_DB}:${envopts[v]}`] = evars[v];
+	Object.entries(evars).forEach((kv) => {
```

Comment:
You can use `([key, val]) =>` and I suggest you give them better names like `envVar, setupKey` or something.

Reply:
ah, cool, decomposing... as I said, `js` noob here, more of a backend/systems guy ;)
Thanks for the info...

Reply:
ok, last commit should address all issues? Thanks for the optimising suggestions...